src/content/en/updates/2017/03/chrome-58-deprecations.md

```diff
@@ -53,6 +53,21 @@ applied. The old behavior was deprecated in Chrome 57, and is now removed.
 [Chromestatus Tracker](https://www.chromestatus.com/feature/5760965337415680) &#124;
 [Chromium Bug](https://bugs.chromium.org/p/chromium/issues/detail?id=659464)
 
+## Remove content-initiated top frame navigations to data URLs
+
+Because of their unfamilliarity to non-technical browser users, we're
+increasingly seeing the `data:` scheme being used in spoofing and phishing
+attacks. To prevent this, we're blocking web pages from loading using `data:`
+from the top frame. This applies to `&lt;a&gt;` tags, `window.open`,
```

Comment:
I suggest rephrasing this as "we're blocking web pages from loading data: URLs in the top frame".